src/tests/fake/subtractions.ts

```diff
@@ -0,0 +1,58 @@
+import { faker } from "@faker-js/faker";
+import nock from "nock";
+import { Subtraction, SubtractionFile, SubtractionMinimal } from "../../js/subtraction/types";
+import { createFakeUserNested } from "./user";
+
+/**
+ * Create a fake subtraction file
+ */
+export function createFakeSubtractionFile(): SubtractionFile {
+    return {
+        download_url: faker.random.word(),
+        id: faker.datatype.number(),
+        name: faker.random.word(),
+        size: faker.datatype.number(),
+        subtraction: faker.random.alphaNumeric(8),
+        type: "fasta",
+    };
+}
+
+/**
+ * Create a fake subtraction
+ */
+export function createFakeSubtraction(subtractionMinimal: SubtractionMinimal): Subtraction {
```

Comment:
Rather than specifying that a subtraction minimal must be passed I would have createFakeSubtraction call `CreateSubtractionMinimal` by default and destructure the returned value. Saves the tester time if they don't want to specify anything about the minimal subtraction.

That said I think it is a good idea to allow overrides for all data-faking testing functions. So imo the  it would be something like this: 
```suggestion
type createFakeSubtractionMinimal = {
    id?: string
    name?: string
    created_at?: date
   // So on and so forth
}


export function createFakeSubtraction(subtractionMinimal: CreateFakeSubtractionMinimal): Subtraction {
```